hack/setup.sh

```diff
@@ -143,8 +143,6 @@ spec:
       - name: pizza-oven
         image: pizza-oven:latest
         imagePullPolicy: IfNotPresent
-        command:
-        - ./pizza-oven
```

Comment:
Makes sense: this was probably missed after we gave the dockerfile a command to run: https://github.com/open-sauced/pizza/commit/583e0386dca98dffd52f0711605f02b036fe3d89